docs/integrations/linkerd/k8s.md

```diff
@@ -157,6 +157,12 @@ Controller](https://github.com/ngrok/kubernetes-ingress-controller) to simplify
 
 To demonstrate how Linkerd and the ngrok Ingress Controller integrate to add additional observability, security, and reliability into your cluster, you'll deploy the [Emojivoto](https://github.com/BuoyantIO/emojivoto) demo application, which was developed by Buoyant, the company that originally developed Linkerd.
 
+1. Create a ngrok static subdomain for ingress if you don't have one already. Navigate to the [**Domains**
```

Comment:
I know it's not consistent through our docs, but the prevailing opinion is to treat ngrok like it starts with a vowel sound, and thus should be "an ngrok".